argonaut/src/main/scala/io/finch/argonaut/package.scala

```diff
@@ -51,5 +51,5 @@ package object argonaut {
    * @return Create a Finch ''EncodeJson'' from an argonaut ''EncodeJson''
    */
   implicit def encodeArgonaut[A](implicit encode: EncodeJson[A]): EncodeResponse[A] =
-    EncodeResponse("application/json")(encode.encode(_).nospaces)
+    EncodeResponse("application/json")(encode.encode(_).nospaces.getBytes("UTF-8"))
```

Comment:
Actually, the `cs` parameter is not considererd to encode bytes into String. The parameter affects only a `Content-Type` header value. Do situations like "I want to return UTF-8 text content, but don't want to set `;charset=utf-8` to `Content-Type` header." exist?

Using `Buf` and omitting `cs` parameter, above code would be:
```scala
def fromString[A](ct: String)(fn: A => String):  EncodeResponse[A] =
    new EncodeResponse[A] {
      def apply(rep: A): Buf = Utf8(fn(rep))
      def contentType: String = ct
      override def charset: Option[String] = Some("UTF-8")
    }
```